blog/2017-02-03-FormulaForMeasuringUnfairness.adoc

```diff
@@ -302,7 +302,7 @@ h|Root squared deviation
 To deal with unequal employees,
 for example if some employees work half as many hours as other employees,
 multiply their number of tasks by the inverse of their FTE (full time equivalent)
-before feeding into this formula's.
+before feeding into this formula.
```

Comment:
Let me know if the intention was different (like _this formula's variables_ or _these formulas_).